cli/daemon/dash/dashapp/src/components/trace/SpanDetail.tsx

```diff
@@ -194,7 +194,10 @@ const NewRequestInfo: FC<{ req: Request; trace: Trace; onStackTrace: (s: Stack)
         <h4 className="text-gray-300 mb-2 font-sans text-xs font-semibold uppercase leading-3 tracking-wider">
           Error
         </h4>
-        <CodeBox error>{decodeBase64(req.err)}</CodeBox>
+        <CodeBox error>
```

Comment:
Have we made these changes for webapp2 too?

Reply:
Not yet, doing that now